labs/05_pom/tests/test_product.py

```diff
@@ -1,25 +1,70 @@
 from playwright.sync_api import Page, expect
-
+from models.add_product import ProductPage
```

Comment:
`add_product` sounds strange for a model page. it would be better `products` or `product_manager`